client/homebrew/editor/snippetbar/snippetsLegacy/classfeature.gen.js

```diff
@@ -32,6 +33,11 @@ module.exports = function(classname){
 		`- **Saving Throws:** ${_.sampleSize(abilityList, 2).join(', ')}`,
 		`- **Skills:** Choose two from ${_.sampleSize(skillList, _.random(4, 6)).join(', ')}`,
 		'',
+		'#### Spellcasting Ability',
+		`**Spell save DC** = ${_.sample([6, 8, 10])} + your proficiency bonus + your ${spellSkill} modifier`,
+		'',
+		`**Spell attack modifier** = your proficiency bonus + your ${spellSkill} modifier  `,
+		'',
```

Comment:
Great work on this PR! Just one thing: The text here needs to be centered in the Legacy version too. In the V3 version we do it with `{{text-align:center`. Here we will need to resort to a raw HTML div: `<div style=text-align:center>`

If you add that for me then I'll merge this PR.

Reply:
Okay, I added that change and made a couple small typo fixes at the same time.